managedtenants/data/selectorsyncset.yaml.j2

```diff
@@ -155,14 +155,14 @@ items:
         name: {{ADDON.metadata['id']}}
         source: addon-{{ADDON.metadata['id']}}-catalog
         sourceNamespace: {{ADDON.metadata['targetNamespace']}}
-        {% if ADDON.get_subscription_config() %}
+        {% if ADDON.get_envs() %}
         config:
-          env:
-          {% for env_obj in ADDON.get_subscription_config().get("env") %}
-          - name: {{env_obj['name']}}
-            value: |-
-              {{env_obj['value']}}
-          {% endfor %}
+            env:
+            {% for env_obj in ADDON.get_envs() %}
+            - name: {{env_obj['name']}}
+              value: |-
+                {{env_obj['value']}}
+            {% endfor %}
```

Comment:
Hey @ashishmax31! Can you confirm the `indentation` here is correct?

Reply:
Thanks, fixed.